commands/geoadd.md

```diff
@@ -1,47 +1,87 @@
 Adds the specified geospatial items (latitude, longitude, name) to the specified
-key. Data is stored into the key as a sorted set, in a way that makes it possible to later retrieve items using a query by radius with the `GEORADIUS` or `GEORADIUSBYMEMBER` commands.
+key.
+Data is stored into the key as a sorted set, in a way that makes it possible
+to later retrieve items using a query by radius with the `GEORADIUS` or
+`GEORADIUSBYMEMBER` commands, or use `GEOSEARCH` searching within rectangular
```

Comment:
@itamarhaber do you think considering we're "deprecating" GEORADIUS* maybe we should only mention GEOSEARCH here? (which has references to GEORADIUS)